docsrc/examples/MCMC Sampling.ipynb

```diff
@@ -151,62 +152,105 @@
     "the console messages, e.g. `bernoulli-201912081451-1.csv`. Output files\n",
     "written to the temporary directory contain an additional 8-character\n",
     "random string, e.g. `bernoulli-201912081451-1-5nm6as7u.csv`."
-   ],
-   "metadata": {}
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "fit"
-   ],
-   "outputs": [],
-   "metadata": {}
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
     "The sampler output files are written to a temporary directory which\n",
     "is deleted upon session exit unless the ``output_dir`` argument is specified.\n",
     "The ``save_csvfiles`` function moves the CmdStan CSV output files\n",
     "to a specified directory without having to re-run the sampler.\n",
     "The console output files are not saved. These files are treated as ephemeral; if the sample is valid, all relevant information is recorded in the CSV files."
-   ],
-   "metadata": {}
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sampler Progress\n",
+    "\n",
+    "Your model make take a long time to fit.  The `sample` method provides two arguments:\n",
+    "    \n",
+    "    - visual progress bar:  `show_progress=True`\n",
+    "    - stream CmdStan ouput to the console - `show_console=True`\n",
+    "    \n",
+    "To illustrate how progress bars work, we will run the bernoulli model. Since the progress bars are only visible while the sampler is running and the bernoulli model takes no time at all to fit, we run this model for 200K iterations, in order to see the progress bars in action."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fit = model.sample(data=data_file, iter_warmup=100000, iter_sampling=100000, show_progress=True)\n"
+   ]
```

Comment:
Note that we need to add tqdm to `docsrc/env.yml`. Even then, I don't think the rendered version of this notebook will contain the progress bars

Reply:
question:  why does `docsrc/env.yml` list `pip` twice?

Reply:
The first `pip` actually installs pip itself, the second starts a list of packages to be installed via pip. The list we use is basically equivalent to running `pip install -e .` in the cmdstanpy root directory